src/Microsoft.TemplateEngine.TemplateLocalizer/Microsoft.TemplateEngine.TemplateLocalizer.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(NETCoreTargetFramework)</TargetFramework>
+    <Description>A tool that can be used to localize template.json files.</Description>
+    <IsPackable>true</IsPackable>
+    <Nullable>enable</Nullable>
+
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>dotnet-template-localizer</ToolCommandName>
+    <LangVersion>9.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0-preview.3.21164.3" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0-preview.3.21164.3" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.21109.1" />
```

Comment:
Same comment as in other .csproj file, lets use `Update` version in Directory.Build.props file on root of repo.